src/components/BeaconsForm.tsx

```diff
@@ -48,11 +52,29 @@ export const BeaconsForm: FunctionComponent<BeaconsFormProps> = ({
             <FormErrorSummary formErrors={formErrors} />
             <Form>
               <FormGroup errorMessages={errorMessages}>
-                <FormFieldset>
-                  <FormLegendPageHeading>{pageHeading}</FormLegendPageHeading>
-                  {pageTextComponent}
-                </FormFieldset>
-                {children}
+                {headingType === "label" ? (
+                  <>
+                    <h1 className="govuk-heading-l govuk-!-margin-bottom-3">
+                      <label htmlFor={id}>{pageHeading}</label>
+                    </h1>
```

Comment:
Looking at the docs for setting a `h1` element as the label heading, it looks like it should be:

```suggestion
                   <h1 className="govuk-label-wrapper">
                      <FormLabel htmlFor={id} className="govuk-label--l">
                        {pageHeading}
                      </FormLabel>
                    </h1>
```


See screenshot and docs from GDS website: https://design-system.service.gov.uk/get-started/labels-legends-headings/